lib/tuxedo.rb

```diff
@@ -82,7 +82,7 @@ def underscored_name
     # alias_method wont work well with inheritance/includes/extends
     #
     def setup_alias_method(name = underscored_name)
-      define_method(name) { object }
```

Comment:
https://github.com/rails/rails/blob/7f18ea14c893cb5c9f04d4fda9661126758332b5/activerecord/lib/active_record/attribute_methods/read.rb#L6 - maybe would be useful to see how rails uses define_method